tests/fixtures/Page.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\CMS\Controllers\ContentController;
+
+class Page extends SiteTree
```

Comment:
Could we at least rename the file in source control to a non-php extension so it doesn't interfere with code inspection? It's going to affect development when Page.php is defined elsewhere.